insights_sha_extractor_test.sh

```diff
@@ -102,7 +102,7 @@ fi
 install_extractor
 
 # shellcheck disable=SC2068
-PYTHONDONTWRITEBYTECODE=1 python3 -m behave --no-capture \
+PYTHONDONTWRITEBYTECODE=1 python3 -m behave -k \
```

Comment:
this is not the same argument. Changed on purpose? What's the output of the execution now?

Reply:
Same output. `--no-capture` is intended to be used when you're trying to debug some step, in order to be able to print the stdout or stderr. `-k` is to show only the steps that are intended to be executed (selected) by the tags selection. With the chosen `--format` output, the `-k` has no effect probably... I'll remove it